src/analysis/object.rs

```diff
@@ -122,6 +133,21 @@ impl Info {
                 format!("prelude::{}", self.trait_name).into(),
                 self.trait_name.as_str().into(),
             ),
+            LocationInObject::VirtualExt | LocationInObject::VirtualExtManual => {
+                // TODO: maybe a different config for subclass trait name?
+                let trait_name = format!("{}Impl", self.trait_name.trim_end_matches("Ext"));
+                (
+                    format!("subclass::prelude::{trait_name}").into(),
+                    trait_name.into(),
+                )
+            }
+            LocationInObject::ClassExt | LocationInObject::ClassExtManual => {
+                let trait_name = format!("{}SubclassExt", self.trait_name.trim_end_matches("Ext"));
```

Comment:
Do we have examples of this other than `ObjectSubclassExt`? What exactly is this good for?

Is this basically like https://gstreamer.pages.freedesktop.org/gstreamer-rs/stable/latest/docs/gstreamer/prelude/trait.ElementClassExt.html , i.e. for class methods? Then the naming here would be wrong

Reply:
we have https://gtk-rs.org/gtk4-rs/git/docs/gtk4/subclass/widget/trait.WidgetClassSubclassExt.html  & https://gtk-rs.org/gtk4-rs/git/docs/gtk4/subclass/cell_area/trait.CellAreaClassSubclassExt.html. So probably the gstreamer one has to be renamed?

Reply:
The GTK4 ones are wrong though. Not everything there is subclass specific and can also be called by other code on existing, external classes. E.g. `gtk_widget_class_query_action()`. So the naming seems wrong and the module it's placed in.

That's the same function pattern as for the GStreamer one. Those functions are called by normal application code.

All the setters in yours are a bit more tricky. Those should probably be in the subclass module, but then we need to somehow distinguish these two cases in gir. OTOH they could as well be outside the subclass module: you can only get a mutable class reference inside `class_init()` so nothing else can call it anyway.